src/pages/LoadingPage.jsx

```diff
@@ -0,0 +1,32 @@
+export function LoadingPage() {
+  return (
+    <div className="fixed inset-0 z-50 flex items-center justify-center bg-black bg-opacity-50">
+      <div className="text-center flex flex-col items-center">
+        <svg
+          className="animate-spin h-12 w-12 mr-3 text-blue-500"
+          xmlns="http://www.w3.org/2000/svg"
```

Comment:
svg ... install material-tailwind and take spinner from it instead of creating spinner with svg